Automattic/newspack-newsletters
# Add background color setting

### All Submissions:






### Changes proposed in this Pull Request:



Add setting for background color. During testing I've found that GMail and Outlook on iOS in dark mode disregard a white background color even if set, there's nothing we can do about it I think. 

Closes #219.

### How to test the changes in this Pull Request:

1. Create a newsletter. Observe the "Typography" panel in sidebar is now named "Styling"
3. Set a background color, observe the editor background updating
3. Send a test email, observe the background color is as set in the settings

### Other information:
